### PR TITLE
add support for next page option on read events

### PR DESCRIPTION
**Pull request by AdriVanHoudt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ Takes options object and an optional callback, either returning a promise for, o
 - **from** - required - the start date as an ISO string.
 - **to** - required - the end date as an ISO string.
 - **tzid** - the timezone id for the query.
+- **next_page** - url for the next page. This will still aplly other options to the request.
```
Comment (warbrett) [inline README.md]: Minor typo on apply.

**Comment by warbrett** [comment]
Awesome! If you want to amend the commit for the typo I'll merge this and bump the version so it gets deployed to npm right away.

**Comment by AdriVanHoudt** [comment]
@warbrett done!

**Comment by AdriVanHoudt** [comment]
:+1:

**Pull request merged**